src/containers/pages/GenericJurisdictionReport/index.tsx

```diff
@@ -200,7 +201,7 @@ const GenericJurisdictionReport = (
 
   let columnsToUse = get(plansTableColumns, jurisdictionColumn, null);
   if (currLevelData && currLevelData.length > 0) {
-    if (currLevelData[0].jurisdiction_depth === +focusAreaLevel) {
+    if (currLevelData[0].jurisdiction_depth >= +focusAreaLevel) {
```

Comment:
This works for now, but we need to make it better.  See: https://github.com/onaio/reveal-frontend/issues/1144